src/pages/index.js

```diff
@@ -2,51 +2,137 @@ import "./index.css";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
-import initialCards from "../utils/constants.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
+import PopupDeleteCard from "../components/PopupDeleteCard.js";
 
 //Buttons
 const profileEditButton = document.querySelector("#profile-edit-btn");
 const addButton = document.querySelector("#add-button");
+const editAvatarButton = document.querySelector(".avatar-edit-button");
 
-function addCard (cardDataObj) {
-  const cardInstance = new Card(cardDataObj, "#card-template", (event) => {
-    imageModalHandler.open(event);
-  });
-  
-  const domCard = cardInstance.returnCard();
-
-  gridHandler.addItem(domCard);
-}
+//modalHandlers
+const deleteModalHandler = new PopupDeleteCard("#delete-card-modal");
```

Comment:
* Please, remove `Handler` from the names of the popups. They are modals - not handlers